src/visualization/ColorPalette/Makefile

```diff
@@ -1,10 +1,13 @@
-main: main.o kmercolor.o Graph.o
-	g++ main.o kmercolor.o Graph.o -o main -lsfml-graphics -lsfml-window -lsfml-system
-main.o: main.cc kmercolor.h Graph.h
+
+all: main
+
+Program: main.o ColorMap.o Graph.o
+	g++ main.o ColorMap.o Graph.o -o Program -lsfml-graphics -lsfml-window -lsfml-system
+main.o: main.cc ColorMap.h Graph.h
 	g++ -c -std=c++11 main.cc
-kmercolor.o: kmercolor.cc kmercolor.h Graph.h
-	g++ -c -std=c++11 kmercolor.cc
-Graph.o: Graph.cpp Graph.h kmercolor.h
+ColorMap.o: ColorMap.cc ColorMap.h Graph.h
+	g++ -c -std=c++11 ColorMap.cc
+Graph.o: Graph.cpp Graph.h ColorMap.h
 	g++ -c -std=c++11 Graph.cpp
 clean:
 	rm *.o main

```

Comment:
```suggestion
	rm -f *.o main
```

Reply:
Otherwise, `make clean` gives an error:
```
make clean
rm *.o main
rm: cannot remove '*.o': No such file or directory
rm: cannot remove 'main': No such file or directory
Makefile:13: recipe for target 'clean' failed
make: *** [clean] Error 1
```